cnf-certification-test/accesscontrol/suite.go

```diff
@@ -700,3 +707,23 @@ func testPodRequestsAndLimits(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+
+func Test1337UIDs(env *provider.TestEnvironment) {
+	// Note this test is only ran as part of the 'extended' test suite.
+	ginkgo.By("Testing pods to ensure none are using UID 1337")
+	const leetNum = 1337
+	var badPods []string
+	for _, put := range env.Pods {
+		ginkgo.By(fmt.Sprintf("checking if pod %s has a securityContext RunAsUser 1337 (ns= %s)", put.Data.Name, put.Data.Namespace))
+		if put.Data.Spec.SecurityContext.RunAsUser != nil && *put.Data.Spec.SecurityContext.RunAsUser == int64(leetNum) {
```

Comment:
Is it a problem if the pointer is nil?

Reply:
Yeah it's a safeguard against segfault.

Reply:
I understand that it prevents null pointer reference. I mean is it en error if the pointer is nil?

Reply:
Oh, nope in fact it's probably better if its nil overall.